.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Please, provide:
+1. Reproducible code snippet
+2. Error traceback
+
```

Comment:
Bcs bug could be not only error, but also some non-expected behaviour, mb let's add section **Behaviour**, so user would be able to show what exactly he's received? I.e. traceback, data duplicates, etc.

Reply:
I meant here, that unexpected behaviour could be hot only error, but real unexpected behaviour. In this case we need both **Behaviour** and **Expected Behaviour** sections. In this case user will be able to explain what he's received and in second - what he's expected to receive. 🙂